### DISCUSSION TITLE
Question 🙋: what should I learn before learning fresh?

### DISCUSSION THREAD

**Discussion by amoktar** [freshframework/fresh#325] [General]
Hello
I'm wondering what should I learn before learning Fresh Framework?

Thanks 🙏

**Comment by lucacasonato** [comment]
HTML, CSS, and TypeScript.

**Reply by nhrones** [reply]
@AMoktar I had little experience with web frameworks, and no experience at all with React/Preact before using Fresh.
I actually used Fresh to learn about these technologies.  I used the starter-app created by Fresh to study/follow the code (including the Fresh source/docs). As I went along, anything I didn't understand, I Googled/studied until I did. 
This is a slow process, but now I feel comfortable with all that is in Fresh.
Good luck! Don't give up! And most of all, Have fun!  Fresh is very user friendly.

**Reply by amoktar** [reply]
@nhrones Thanks for this lovely encouraging reply.
I wish you good luck with everything 💐💐💐

**Reply by nhrones** [reply]
Your welcome.  I have a few fresh apps you might want to look at  -  `https://github.com/nhrones`
